dashboard/app/views/levels/_standalone_video.html.haml

```diff
@@ -0,0 +1,18 @@
+:javascript
+  function getResult() {
+    return {
+      response: 'ok',
+      result: true
+    };
+  }
+
+.standalone-video
+  - video = @level.specified_autoplay_video
+  - src = "/videos/embed/#{video.youtube_code}?rel=0"
+  %h2= video.localized_name
+  %div= render(inline: @level.markdown_instructions, type: :md) if @level.markdown_instructions
+
+  %iframe{width: 853, height: 480, frameborder: 0, allowfullscreen: true, src: src}
```

Comment:
If you can define this in CSS as some percentage of the normal viewport, you can remove these magic numbers (at least the width, maybe not the height)